ethsigner/signer/hashicorp/src/main/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSubCommand.java

```diff
@@ -93,6 +112,14 @@ public String getCommandName() {
     return COMMAND_NAME;
   }
 
+  public boolean isTlsEnabled() {
+    return Optional.ofNullable(tlsEnabled).orElse(false);
```

Comment:
No, it will always have a value, this remained from an earlier attempt where I used "boolean" instead of negateable option. I will remove the nullable bit.

Reply:
fixed. return variable directly.